src/analysis/normed_space/weak_dual.lean

```diff
@@ -149,36 +148,4 @@ begin
   exact set.preimage_set_of_eq,
 end
 
-variables (𝕜)
-
-/-- The polar set `polar 𝕜 s` of `s : set E` seen as a subset of the dual of `E` with the
-weak-star topology is `weak_dual.polar 𝕜 s`. -/
-def polar (s : set E) : set (weak_dual 𝕜 E) := to_normed_dual ⁻¹' (polar 𝕜 s)
-
 end weak_dual
-
-end weak_star_topology_for_duals_of_normed_spaces
-
-section polar_sets_in_weak_dual
-
-open metric set normed_space
-
-variables {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
-variables {E : Type*} [normed_group E] [normed_space 𝕜 E]
-
-/-- The polar `polar 𝕜 s` of a set `s : E` is a closed subset when the weak star topology
-is used, i.e., when `polar 𝕜 s` is interpreted as a subset of `weak_dual 𝕜 E`. -/
-lemma weak_dual.is_closed_polar (s : set E) : is_closed (weak_dual.polar 𝕜 s) :=
```

Comment:
Is the replacement for this statement ready to go in another PR?

Reply:
it is in this PR now